Makefile

```diff
@@ -49,25 +52,42 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 
 .PHONY: fmt
 fmt: ## Run go fmt against code.
-	go fmt ./...
+	$(GO_FMT) ./...
```

Comment:
Now this doesn't work :)

It looks like `go fmt` is not simply an alias for `gofmt -l -w`. It internally lists all the files in the package. I guess leave it as `$(GO_CMD) fmt`

Reply:
back to GO_CMD